herddb-docker/README.md

```diff
@@ -0,0 +1,59 @@
+# How to use this image
+
+## Running single instance HerdDB
```

Comment:
can you please add a paragraph about how to build the image ?

Reply:
i'm working on the documentation, and will put those instruction on the next pull request.